### DISCUSSION TITLE
You've been a pain in the ass with your politics messages...

### DISCUSSION THREAD

**Discussion by dardizzola** [shaked6540/YoutubePlaylistDownloader#308] [General]
I believe no Israeli or Palestinian authorities frequent GitHub.
You've been a pain in the ass with your messages...
This is a work environment where we talk about technology, not politics!

**Comment by shaked6540** [comment]
Says the guy who stole the entire code and tried to rebrand it, while removing all credits of me and the translators

**Comment by cristianst85** [comment]
Hey mister _italiano_, is not the maintainer who opens these _issues_.
Did your brain melted because of the heat wave?
You also stole from me. 
Hai, sictir!

**Comment by Amyrosie** [comment]
> I believe no Israeli or Palestinian authorities frequent GitHub. You've been a pain in the ass with your messages... This is a work environment where we talk about technology, not politics!






I logged in just to let you know you're a goddamn idiot. 
It's because of people like you that we can't have nice things. Keep your crappy opinions to yourself and move on. No idea why you thought it was intelligent to make a post about this. Butthurt much ?

**Comment by n3td3vofficial** [comment]
**I STAND WITH ISRAEL!**

_Genesis 12:1-3 KJB_
1 Now the Lord had said unto Abram, Get thee out of thy country, and from thy kindred, and from thy father's house, unto a land that I will shew thee:
2 And I will make of thee a great nation, and I will bless thee, and make thy name great; and thou shalt be a blessing:
3 And I will bless them that bless thee, and curse him that curseth thee: and in thee shall all families of the earth be blessed.

**Comment by AbdulrahmanHashem** [comment]
what do you expect from a guy that supports the mass killing of innocent people, ofcourse he is gonna express his genocidal mentality.
notice that the people who responded are all mentally deranged, they are making the stereotype themselves XD

I only post because i want to say i stand with israel

**Reply by shaked6540** [reply]
thanks for the support
